Microsoft.Toolkit.Uwp.UI/Media/RadialGradientBrush.cs

```diff
@@ -103,5 +106,35 @@ protected override bool OnDraw(CanvasDevice device, CanvasDrawingSession session
 
             return false;
         }
+
+        private void RegisterAppResumeHandler()
+        {
+            Device.DeviceLost += RadialGradientBrush_DeviceLost;
+            Graphics.RenderingDeviceReplaced += RadialGradientBrush_RenderingDeviceReplaced;
+        }
+
+        private void RadialGradientBrush_RenderingDeviceReplaced(CompositionGraphicsDevice sender, Windows.UI.Composition.RenderingDeviceReplacedEventArgs args)
+        {
+            Graphics.RenderingDeviceReplaced -= RadialGradientBrush_RenderingDeviceReplaced;
```

Comment:
Is this the right place to call this? If the device is lost or the rendering device is replaced, you will unsubscribe from the events, but they get subscribed only at the ctor. They will never be re-subscribed. This logic seems flawed for me.